common/persistence/nosql/nosqlplugin/cassandra/workflow_utils.go

```diff
@@ -1695,22 +1695,30 @@ func mustConvertToSlice(value interface{}) []interface{} {
 	}
 }
 
-func populateGetReplicationTasks(query gocql.Query) ([]*nosqlplugin.ReplicationTask, []byte, error) {
+func populateGetReplicationTasks(query gocql.Query) ([]*nosqlplugin.HistoryMigrationTask, []byte, error) {
 	iter := query.Iter()
 	if iter == nil {
 		return nil, nil, &types.InternalServiceError{
 			Message: "populateGetReplicationTasks operation failed.  Not able to create query iterator.",
 		}
 	}
 
-	var tasks []*nosqlplugin.ReplicationTask
+	var tasks []*nosqlplugin.HistoryMigrationTask
 	task := make(map[string]interface{})
 	for iter.MapScan(task) {
 		t := parseReplicationTaskInfo(task["replication"].(map[string]interface{}))
+		taskID := task["task_id"].(int64)
+		data := task["data"].([]byte)
+		encoding := task["data_encoding"].(string)
+		taskBlob := persistence.NewDataBlob(data, common.EncodingType(encoding))
 		// Reset task map to get it ready for next scan
 		task = make(map[string]interface{})
 
-		tasks = append(tasks, t)
+		tasks = append(tasks, &nosqlplugin.HistoryMigrationTask{
+			Replication: t,
+			Task:        taskBlob,
+			TaskID:      taskID,
+		})
```

Comment:
this snipped exists in multiple places. does it make sense to move it to a helper?